rpp30-boc-heartbleed/whimsical
# create and style navbar, insert in login component to test

## Description
created navbar using fontawesome icons. It is currently nested in the login component for testing purposes.

## How to test
upon running server and loading in simulator, login screen should appear with navbar at bottom of screen. click on respective icons for userprofile, friendslist, and dashboard to test navigation functionality.

## Notes
note that I modified the styling of the styles.container to fit the navbar at the bottom of the screen, as well as added additional styling to the <View> containing the navbar in the login component. As other components get fleshed out, the Navbar component will have to be nested accordingly (note how props are currently passed into the Navbar from the Login component) and container styling adjusted.

## Issue tracking
(https://trello.com/c/w8oYLqiw/31-create-navbar)
